bitnami/discourse/README.md

```diff
@@ -186,8 +187,9 @@ The following table lists the configurable parameters of the Discourse chart and
 |-------------------------------------------|---------------------------------------------------------------------------------------|--------------------------------------------------------------|
 | `postgresql.enabled`                      | Deploy PostgreSQL container(s)                                                        | `true`                                                       |
 | `postgresql.postgresqlUsername`           | PostgreSQL user to create (used by Discourse)                                         | `bn_discourse`                                               |
-| `postgresql.postgresqlPassword`           | Password for the Dicourse user                                                        | _random 10 character long alphanumeric string_               |
-| `postgresql.postgresqlPostgresPassword`   | Password for the admin user ("postgres")                                              | `bitnami`                                                    |
+| `postgresql.postgresqlPassword`           | Password for the Dicourse user - ignored if existingSecret is set to true                                                       | _random 10 character long alphanumeric string_               |
```

Comment:
Could you please re-word this?

...ignored if existingSecret _is set to true_ --> ignored if existingSecret _is provided_

Same for the `postgresql.postgresqlPostgresPassword` description, as existingSecret is not a boolean.